SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowController.cpp

```diff
@@ -85,28 +65,32 @@ void ASpatialFunctionalTestFlowController::CrossServerSetWorkerId_Implementation
 	WorkerDefinition.Id = NewWorkerId;
 }
 
-void ASpatialFunctionalTestFlowController::OnReadyToRegisterWithTest()
-{
-	TryRegisterFlowControllerWithOwningTest();
-}
-
 void ASpatialFunctionalTestFlowController::OnRep_OwningTest()
 {
-	TryRegisterFlowControllerWithOwningTest();
+	// Register replicated flow controllers
+	RegisterFlowController();
```

Comment:
This can probably happen multiple times - is that okay?

Reply:
hmm, maybe it cannot happen multiple times...

Reply:
Currently, it does not occur multiple times and there is a check to ensure it is not called multiple times in the future.